blog/2021/02/julia-1.6-highlights.md

```diff
@@ -0,0 +1,389 @@
+
+Our work to slim JLL packages down resulted in the creation of a new package,
+[`JLLWrappers.jl`](https://github.com/JuliaPackaging/JLLWrappers.jl).  This
+package provides macros that auto-generate the bindings necessary for a JLL
+package, and do so by using the minimum number of functions and datastructures
+possible.  By limiting the number of backedges and datastructures, as well as
+centralizing the template pieces of code that each JLL package uses, we are
+able to not only vastly improve load times, but improve compile times as well!
+As an added bonus, improvements to JLL package APIs can now be made directly in
+`JLLWappers.jl` without needing to re-deploy hundreds of JLLs.  Because these
+JLL packages only define a thin wrapper around simple, lightweight functions
+that load libraries and return paths and such, they do not benefit from the
+heavy optimization that most Julia code undergoes.  One final piece of the
+optimization puzzle was therefore to disable optimizations and use the new
+[per-module optimization levels](https://julialang.org/blog/2020/08/julia-1.5-highlights/#per-module_optimization_levels)
+functionality to reduce the amount of time spent generating a very small
+amount of code, saving precious seconds.
+
+~~~
+<iframe width="100%" height="450" frameborder="0" scrolling="no" src="https://chart-studio.plotly.com/~staticfloat/76.embed"></iframe>
+~~~
+
+The interplay between compiler improvements and the benefits that `JLLWrappers`
+affords were [well-recorded](https://github.com/JuliaGraphics/Gtk.jl/issues/466#issuecomment-716058685)
+during the development process, and showcase a speedup of load times for the
+original, non-JLLWrapperized `GTK3_jll` package from its peak at `6.73` seconds
+on Julia v1.4 down to `2.34` seconds on Julia v1.6, purely from compiler
+improvements.  If you haven't thanked your local compiler team today, you
+probably should.  Using the slimmed-down JLLWrappers implementation of all
+relevant JLLWrappers packages results in a further lowering of load time down
+to a blistering `140ms`.  End-to-end, this means that this work effected a 
+roughly **`50x` speedup** in load times for large trees of binary artifacts.
+While there are some minor improvements for lazy loading of shared libraries
+and such in the pipeline, we are confident that this work will provide a strong
+foundation for Julia's binary packaging story for the forseeable future.
+
+## Downloads & NetworkingOptions
+
+_Stefan Karpinski_
+
+In previous releases, when you download something in Julia, either directly,
+using the `Base.download` function, or indirectly when using `Pkg`, the actual
+downloading was done by some external process—whichever one of `curl`, `wget`,
+`fetch` or `PowerShell` happened to be available on your system. The fact that
+this frankendownload feature worked at all was something of a miracle, that only
+worked due to much fussy command-line-option finessing of over the years. And
```

Comment:
```suggestion
worked due to much fussy command-line-option finessing over the years. And
```

Reply:
`of` seems fine. The comma, perhaps. Your call @logankilpatrick